app/backend/approaches/retrievethenread.py

```diff
@@ -68,6 +52,36 @@ def __init__(
         self.query_speller = query_speller
         self.chatgpt_token_limit = get_token_limit(chatgpt_model, self.ALLOW_NON_GPT_MODELS)
 
+        self._initialize_templates()
+
+    def _initialize_templates(self):
+        self.env = Environment(loader=FileSystemLoader('approaches/prompts/ask'))
+        self.system_chat_template = self.env.get_template('system_message.jinja').render()
+        self.few_shots = self._process_few_shots()
+    
+    def _process_few_shots(self) -> Any:
```

Comment:
Given the amount of processing here, I'd be inclined to leave the few shots as .json, but inside the prompts folder, and just load with json package.

Reply:
Same here, I’ve converted the few_shots examples to JSON.